docs/index.rst

```diff
@@ -3,13 +3,13 @@ The Littlest JupyterHub
 =======================
 
 A simple `JupyterHub <https://github.com/jupyterhub/jupyterhub>`_ distribution for
-a small (0-50) number of users on a single server.
+a small (0-100) number of users on a single server.
```

Comment:
YES! Have to decide what's the number we should use. 100 seems ok for now.

Reply:
Wow thats awesome!!! :D